iothub/service/src/Jobs/JobRequest.cs

```diff
@@ -26,39 +26,52 @@ public class JobRequest
         public JobType JobType { get; set; }
 
         /// <summary>
-        /// Required if jobType is cloudToDeviceMethod.
         /// The method type and parameters.
         /// </summary>
+        /// /// <remarks>
+        /// Required if jobType is cloudToDeviceMethod.
+        /// </remarks>
         [JsonProperty(PropertyName = "cloudToDeviceMethod")]
         public CloudToDeviceMethod CloudToDeviceMethod { get; set; }
 
         /// <summary>
-        /// Required if jobType is updateTwin.
         /// The Update Twin tags and desired properties.
         /// </summary>
+        /// <remarks>
+        /// Required if the job type is UpdateTwin.
+        /// </remarks>
         [JsonProperty(PropertyName = "updateTwin")]
         public Twin UpdateTwin { get; set; }
 
         /// <summary>
-        /// Required if jobType is updateTwin or cloudToDeviceMethod.
-        /// Condition for device query to get devices to execute the job on
+        /// Condition for device query to get devices to execute the job on.
         /// </summary>
+        /// <remarks>
+        /// Required if job type is UpdateTwin or CloudToDeviceMethod.
+        /// </remarks>
         [JsonProperty(PropertyName = "queryCondition")]
         public string QueryCondition { get; set; }
 
 
         /// <summary>
-        /// ISO 8601 date time to start the job
+        /// ISO 8601 date time to start the job.
         /// </summary>
-        [JsonProperty(PropertyName = "startTime")]
+        [JsonProperty(PropertyName = "startTimeUtc")]
```

Comment:
You can't change the serialized value. The service still expects `startTime`.

The only rename was meant to be to the .NET property.